<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wSimulation</class>
 <widget class="QWidget" name="wSimulation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pbStartSim">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="3" colspan="2">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pbResults">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Результаты</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
